tests/integration/test_coverage_lift_to_75.py

```
@@ -0,0 +1,634 @@
+
+    def test_resolve_conflict_increment_strategy_at_area_level(self) -> None:
+        # Hits lines 462-465 (increment strategy, area-level branch).
+        # Area-level resolve dispatches to generate_area_number, which honors
+        # the preferred-area-free invariant: the bare area number is occupied
+        # but categories under it are still free, so the resolver returns a
+        # JohnnyDecimalNumber with the same area and the original name.
+        from methodologies.johnny_decimal.categories import JohnnyDecimalNumber
+
+        _, gen = self._make()
+        conflict = JohnnyDecimalNumber(area=10, name="X", description="d")
+        gen.register_existing_number(conflict, Path("/tmp/x"))
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/x"

(S108)